iidm/iidm-mergingview/src/main/java/com/powsybl/iidm/mergingview/BusAdapter.java

```diff
@@ -32,189 +33,190 @@ protected BusAdapter(final Bus delegate, final MergingViewIndex index) {
     }
 
     @Override
-    public ComponentAdapter getSynchronousComponent() {
-        return getIndex().getComponent(getDelegate().getSynchronousComponent());
+    public Iterable<TwoWindingsTransformer> getTwoWindingsTransformers() {
+        return getTwoWindingsTransformerStream().collect(Collectors.toSet());
     }
 
     @Override
-    public ComponentAdapter getConnectedComponent() {
-        return getIndex().getComponent(getDelegate().getConnectedComponent());
+    public Stream<TwoWindingsTransformer> getTwoWindingsTransformerStream() {
+        return getDelegate().getTwoWindingsTransformerStream().map(getIndex()::getTwoWindingsTransformer);
     }
 
-    // -------------------------------
-    // Simple delegated methods ------
-    // -------------------------------
-
-    // -------------------------------
-    // Not implemented methods -------
-    // -------------------------------
     @Override
-    public VoltageLevelAdapter getVoltageLevel() {
-        throw MergingView.NOT_IMPLEMENTED_EXCEPTION;
+    public ComponentAdapter getSynchronousComponent() {
+        return getIndex().getComponent(getDelegate().getSynchronousComponent());
```

Comment:
Is this implementation ok? synchronous and connected component have to be merged due to merged lines

Reply:
This will be done later, to be reverted